core/sass/marketaccess.scss

```diff
@@ -66,4 +77,23 @@ form {
         margin-top: 40px;
         display: block;
     }
+}
+
+#id_problem-details-product_service-container {
+    .form-label {
+        padding-bottom: 0.25em;
```

Comment:
ems are nice if you have a responsive font size but usually you want to stick to our designers' predefined spacings in px. they're in `partials/vars` - 15px, 30px, 45px, 60px, 90px. though if you need just a small amount of space using an arbitrary px or em value is fine